### PR TITLE
PF Issue 505 - Required Field & Help On Forms Design Updates

### PR DESCRIPTION
**Pull request by nlcwong**
## Description
Updating designs for Field Level Help icon and Required Field Symbol placement so that they do not clash. Github Issue Link: https://github.com/patternfly/patternfly-design/issues/505

Closes: #505 

## Changes
* Field Labeling - moved the required field symbol from being to the right of the label to now residing to the left of the field label.
* Field Level Help - changed the color of the field level help icon (info icon) to be grey and on hover over, change to blue.
* Updated overview (and in some cases created design) pages for field labeling and help on forms to reflect changes.

## Contribution Submission Checklist
- [x] A GitHub issue was created and reviewed with the PatternFly community.
- [x] The pattern design overview.md and design.md templates were used to provide all information, relevant to this design.
- [x] Any new files have been added to the folder for that pattern.
- ~~[ ] The folder, for brand new patterns, has been titled with the name of the pattern.~~
- [x] All images are in an "img" folder and embedded images have the correct filepath.
- [x] The overview.md, design.md and site.md files, for this design have been commited and pushed to the patternfly-design repository, for review.

### CODE REVIEW

**Comment by nlcwong** [comment]
@mcarrano I made the changes you asked for, please review (hopefully I did that right, i edited the file from online and then committed it to the branch ... cross those fingers).  Let me know if i need to make more changes or do it differently.

File: pattern-library/forms-and-controls/field-labeling/design/overview.md [hunk 1]
```diff
@@ -2,37 +2,40 @@
...
 
-## High-Level Recommendations
-
-* Make [right aligned labels](#right-aligned-labels) should be your default choice.  It has been shown that this approach leads to the shortest completion times.
 
-* Use [top aligned labels](#top-aligned-labels) when you are space constrained or if mobile usage is a high priority.
+## High-Level Recommendations
 
+* Use [right-aligned labels](#right-aligned-labels) as your default choice.  It has been shown that this approach leads to the shortest completion times.
+* Use [top-aligned labels](#top-aligned-labels) when you are space constrained or if mobile usage is a high priority.
```
Comment (catrobson) [inline pattern-library/forms-and-controls/field-labeling/design/overview.md]: Right and Top aligned are represented here, but not left aligned.  Is that on purpose?

Comment (mcarrano) [reply pattern-library/forms-and-controls/field-labeling/design/overview.md]: To @nlcwong 's point, I think the original intent was to steer people towards right aligned labels as the default recommendation. Do you still think that makes sense @catrobson ?

Comment (catrobson) [reply pattern-library/forms-and-controls/field-labeling/design/overview.md]: Fine with me @mcarrano

**Comment by mcarrano** [comment]
@nlcwong Looks good. Thanks again for all your effort on this! There does seem to be some type of conflict that is preventing me from merging this. @dgutride can you take a look at this? I don't understand why Rebase an Merge is being blocked.

**Pull request merged**